osgp/protocol-adapter-dlms/osgp-protocol-adapter-dlms/src/main/java/org/opensmartgridplatform/adapter/protocol/dlms/domain/commands/DlmsHelperService.java

```diff
@@ -179,6 +178,23 @@ public DataObject getClockDefinition() {
         return DataObjectDefinitions.getClockDefinition();
     }
 
+    public DataObject getAccessSelectionTimeRangeParameter(final DateTime beginDateTime, final DateTime endDateTime,
+            final DataObject selectedValues) {
+
+        /*
+         * Define the clock object {8,0-0:1.0.0.255,2,0} to be used as
+         * restricting object in a range descriptor with a from value and to
+         * value to determine which elements from the buffered array should be
+         * retrieved.
+         */
+        final DataObject clockDefinition = this.getClockDefinition();
+
+        final DataObject fromValue = this.asDataObject(beginDateTime);
+        final DataObject toValue = this.asDataObject(endDateTime);
```

Comment:
Notice that begin -> from and end -> to. This is confusing, I suggest to use 'from' and 'to' throughout. You can work around a potential name clash by inlining the intermediary variables or extracting a method or making it clear in the variable name what the difference is between a `DateTime from` and a `DataObject from`.

Reply:
Aangepast